--- ../upstream/bug16946.phpt
+++ bug16946.phpt
@@ -42,20 +42,20 @@ array(2) {
   string(4) "4567"
 }
 string(%d) "GET /test HTTP/%f
-User-Agent: PECL-OAuth/%f%s
 Host: 127.0.0.1:12342
+User-Agent: PECL-OAuth/%f%s
 Accept: */*
 Authorization: OAuth oauth_consumer_key="1234",oauth_signature_method="HMAC-SHA1",oauth_nonce="%s.%d",oauth_timestamp="%d",oauth_version="1.0",oauth_signature="%s"
 
 GET /some_url_that_goes_nowhere_and_could_be_very_long.html?bla=bla&mekker=mekker HTTP/%f
-User-Agent: PECL-OAuth/%f%s
 Host: 127.0.0.1:12342
+User-Agent: PECL-OAuth/%f%s
 Accept: */*
 Authorization: OAuth oauth_consumer_key="1234",oauth_signature_method="HMAC-SHA1",oauth_nonce="%s.%d",oauth_timestamp="%d",oauth_version="1.0",oauth_signature="%s"
 
 GET /some_other_url.html HTTP/%f
-User-Agent: PECL-OAuth/%f%s
 Host: 127.0.0.1:12342
+User-Agent: PECL-OAuth/%f%s
 Accept: */*
 Authorization: OAuth oauth_consumer_key="1234",oauth_signature_method="HMAC-SHA1",oauth_nonce="%s.%d",oauth_timestamp="%d",oauth_version="1.0",oauth_signature="%s"
 
